### DISCUSSION TITLE
Achievements visibility issue

### DISCUSSION THREAD

**Discussion by azuloo** [community/community#64608] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,

I've recently toggled (turned it off) "Include private contributions on my profile" option and under my Profile it says "You unlocked new Achievements with private contributions! ..." .
When I turned it back on - there were no achievements displayed in my Profile page. I have "Show Achievements on my profile" enabled so it's not the issue.
I've also committed 2 PR's recently (to public repos) which have been merged and I haven't got "Pull Shark" either.

Am I missing something here?
Thanks

**Comment by ShonChassel** [comment]
When you toggle off "Include private contributions on my profile," your private contributions (commits, pull requests, etc.) won't be visible on your public profile. However, GitHub still acknowledges your private contributions privately and might award achievements based on them. This is why you saw the "You unlocked new Achievements with private contributions!" message.

**Reply by azuloo** [reply]
Yes, it makes sense. But my question was not about why my private commits weren't shown on my activity graph. It's about achievements that are missing even after I've done some steps to get them.

**Comment by azuloo** [comment]
They suddenly appeared after a couple of more PRs. The issue still remains unclear but I'm closing this one as it's resolved for me.
